src/diffusers/pipelines/pipeline_utils.py

```diff
@@ -1346,6 +1407,10 @@ def download(cls, pretrained_model_name, **kwargs) -> Union[str, os.PathLike]:
             allow_patterns += [f"{custom_pipeline}.py"] if f"{custom_pipeline}.py" in filenames else []
             # also allow downloading config.json files with the model
             allow_patterns += [os.path.join(k, "config.json") for k in model_folder_names]
+            # also allow downloading the dduf
+            # TODO: check that the file actually exist
+            if dduf is not None:
+                allow_patterns += [dduf]
```

Comment:
Should `dduf` argument values go through some validations?

Reply:
Yeah i left it as a todo above but we would have to check that the file do actually exist. Any other suggestion is welcomed.

Reply:
Then perhaps a comment of TODO would be nice.